src/pages/ShowArticle.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useEffect, useState } from "react";
+import {
+  IonHeader,
+  IonTitle,
+  IonToolbar,
+  IonContent,
+  IonItem,
+  IonButton,
+  IonPage,
+  IonText
+} from "@ionic/react";
+import { connect } from "react-redux";
+import axios from "axios";
+
+const ShowArticle = props => {
+  let [currentArticle, setCurrentArticle] = useState("");
+
+  const getCurrentArticle = async (id, language) => {
+    try {
+      const response = await axios.get(`/articles/${id}`, {
+        params: { locale: language }
+      });
+      setCurrentArticle(response.data.article);
+    } catch (error) {
+      if (error.message === "Network Error") {
+        props.changeMessage(error.message);
+      } else {
+        props.changeMessage(error.response.data.response);
+      }
+    }
+  };
+
+  useEffect(() => {
+    getCurrentArticle(props.currentArticleId, props.language);
+  }, [props.currentArticleId]);
+
+  return (
+    <IonPage>
+      <IonHeader>
+        <IonToolbar>
+          <IonTitle id="header" routerLink="/home">
+            The Reactive Herald{" "}
+          </IonTitle>
+        </IonToolbar>
+      </IonHeader>
+      {currentArticle && (
+        <IonContent>
+          <IonItem class="show-article">
+            <IonText>
+              <h3>{currentArticle.title}</h3>
+              {currentArticle.body}
+              <img src={currentArticle.image}></img>
```

Comment:
perhaps say that you are not showing the whole article body because the user is not a subscriber if that is the case

Reply:
Fixed